sakura_core/typeprop/CImpExpManager.cpp

```diff
@@ -766,10 +765,10 @@ bool CImpExpKeyHelp::Import( const wstring& sFileName, wstring& sErrMsg )
 		WCHAR *p1, *p2, *p3;
 		p1 = &buff[9];
 		p3 = p1;					//結果確認用に初期化
-		if( NULL != (p2=wcsstr(p1,LTEXT(","))) ){
+		if( NULL != (p2=wcschr(p1,LTEXT(','))) ){
```

Comment:
```suggestion
		if( p2 = wcschr(p1, L',') ){
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17